devito/types/basic.py

```diff
@@ -335,7 +335,7 @@ class Symbol(AbstractSymbol, Cached):
     """
 
     @classmethod
-    def _cache_key(cls, *args, **kwargs):
+    def _make_key(cls, *args, **kwargs):
```

Comment:
Note: renamed it but only classmethod here not part of caching.

Reply:
ok, but then move it below `__new__` ?